README.md

```diff
@@ -507,7 +507,10 @@ All the configurations are grouped in *sections* and some of them can vary depen
 
 - `[Classifier]`:
 
-  Some of the supported devices includes the *facemask detection* feature. If you want to include this feature, you need to specify the classifier section.
+  Some of the supported devices include models that allow for detecting the body-skeleton of a person.
+  This is a key component to **Facemask Detection**.
+  If you want to include this feature, you need to uncomment this section, and use a model that supports the Classifier.
+  Else you can delete or uncomment this section of the config file to save on CPU usage.
```

Comment:
```suggestion
  Otherwise, you can delete or uncomment this section of the config file to save on CPU usage.
```

or something similar. I think `Else` is not used like in programming